src/Calculations/Addition.cs

```diff
@@ -12,16 +11,21 @@ public Addition(IUserConsoleInputValidation validateInput)
     public double FirstNumber { get; private set; }
     public double SecondNumber { get; private set; }
 
-    public void AddWorkFlow()
+    public double Num1()
```

Comment:
it's not clear from a name what `Num1` method is doing. Method name must have a verb or action